packages/grid/src/vaadin-grid-column.js

```diff
@@ -822,6 +822,18 @@ class GridColumn extends ColumnBaseMixin(DirMixin(PolymerElement)) {
         value: false,
       },
 
+      /**
+       * When true, `tabindex` attribute is set on the element with `role="button"` inside a cell,
+       * rather than cell itself. This is needed to keep focus in sync with VoiceOver cursor when
+       * navigating with Control + Option + arrow keys: focusing the `<td>` element does not fire
+       * `focus` event, but focusing an element with `role="button"` inside a cell fires it.
```

Comment:
Suggestion:
> When true, wraps the cell's slot into an element with `role="button"`, and sets the `tabindex` attribute on the button element, instead of the cell itself. This is needed to keep focus in sync with VoiceOver cursor when navigating with Control + Option + arrow keys: focusing the `<td>` element does not fire a `focus` event, but focusing an element with `role="button"` inside a cell fires it.

Reply:
Done.